app/models/service/containers.rb

```diff
@@ -24,6 +24,10 @@ def self.start_container!(wizard_state)
       end
     end
 
+    def self.pull_image(container)
+      container.compute_resource.create_image(:fromImage => container.repository_pull_url)
```

Comment:
Perhaps the issue should be rephrased to make it clear that we are dealing with a bug related to standalone registries only.